src/modules/currency-converter/services/currency.converter.compute.service.ts

```diff
@@ -0,0 +1,150 @@
+import { Constants, ErrorLoggerAsync } from '@multiversx/sdk-nestjs-common';
+import { ApiService } from '@multiversx/sdk-nestjs-http';
+import { Injectable } from '@nestjs/common';
+import { GetOrSetCache } from 'src/helpers/decorators/caching.decorator';
+import {
+    CurrencyCategory,
+    CurrencyRateModel,
+    CurrencyRateType,
+} from '../models/currency.rate.model';
+import { ApiConfigService } from 'src/helpers/api.config.service';
+import { TokenComputeService } from 'src/modules/tokens/services/token.compute.service';
+import BigNumber from 'bignumber.js';
+import { cryptoRatesIdentifiers, mxConfig, tokenProviderUSD } from 'src/config';
+import { TokenService } from 'src/modules/tokens/services/token.service';
+
+@Injectable()
+export class CurrencyConverterComputeService {
```

Comment:
rename into `CurrencyConverterService`. There is no computing performed in this service.